test/intl402/Intl/getCanonicalLocales/returned-object-is-an-array-with-frozen-elements.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-intl.getcanonicallocales
+description: Tests that the value returned by getCanonicalLocales is an Array
+with each element frozen that can be expanded.
```

Comment:
this breaks the front-matter parser.

If you want to provide a multi-line description, try the following format:

```yaml
description: >
  Tests that the value returned by getCanonicalLocales is an Array
  with each element frozen that can be expanded.
```